### ISSUE TITLE
Messages were able to be sent but not received, got a missing session information error upon restarting Element

### ISSUE THREAD

**Issue by ArrayBolt3** [vector-im/element-web#24478]
### Steps to reproduce

I don't actually know how I got Element into this state. I have quite a few Chrome tabs open (about 50+) though, and have Element installed as a PWA. The chat broke sometime shortly after taking my laptop out of sleep, and the error message was shown after restarting Element.

#### What did you expect?

Element Web should always remain able to send and receive messages, and should never have one half of the communication channel break.

#### What happened instead?

After taking my laptop out of sleep and attempting to send messages into a Libera.Chat room, I could see the messages arriving on the IRC side, but messages sent to me (even by people with Matrix.org accounts) weren't showing up in Element. Upon closing and restarting Element, I was shown a "Missing session information" error message.

### Operating system

Ubuntu Unity 22.10

### Browser information

Google Chrome 110.0.5481.77 (Official Build) (64-bit)

### URL for webapp

app.element.io

### Application version

Element version: 1.11.21
Olm version: 3.2.12

### Homeserver

matrix.org

### Will you send logs?

Yes

**Comment by ArrayBolt3** [comment]
I just realized only after sending this report that I had cleared some (but not all) of my cookies and browser cache way earlier today. This is almost certainly what triggered the problem. I assume this is a case of user error and can't be fixed in Element (it's not Element's fault anyway), but just in case, I'll leave this open and leave it to a developer to close it. Sorry if this ended up being useless noise.
